packages/server/src/threads/automation.ts

```diff
@@ -310,87 +310,83 @@ class Orchestrator {
   }
 
   async execute(): Promise<AutomationResults> {
-    return tracer.trace(
-      "Orchestrator.execute",
-      { resource: "automation" },
-      async span => {
-        span?.addTags({ appId: this.appId, automationId: this.automation._id })
-
-        const job = cloneDeep(this.job)
-        delete job.data.event.appId
-        delete job.data.event.metadata
-
-        if (this.isCron() && !job.data.event.timestamp) {
-          job.data.event.timestamp = Date.now()
-        }
-
-        const trigger: AutomationTriggerResult = {
-          id: job.data.automation.definition.trigger.id,
-          stepId: job.data.automation.definition.trigger.stepId,
-          inputs: null,
-          outputs: job.data.event,
-        }
-        const result: AutomationResults = { trigger, steps: [trigger] }
-
-        const ctx: AutomationContext = {
-          trigger: trigger.outputs,
-          steps: [trigger.outputs],
-          stepsById: {},
-          stepsByName: {},
-          user: trigger.outputs.user,
-        }
-        await enrichBaseContext(ctx)
+    return await tracer.trace("execute", async span => {
+      span.addTags({ appId: this.appId, automationId: this.automation._id })
```

Comment:
I don't know why GitHub is being so weird about this diff, it should just be an indentation change.